api/src/paths/project/{projectId}/survey/{surveyId}/update.ts

```diff
@@ -442,3 +454,19 @@ export const updateSurveyProprietorData = async (
     throw new HTTP409('Failed to update survey proprietor data');
   }
 };
+
+export const updateSurveyPermitNumber = async (survey_id: number, connection: IDBConnection): Promise<boolean> => {
```

Comment:
nit: can you update this function name to also be more descriptive, similar to the SQL function